### PR TITLE
ci: Generate dummy keystore in PR builds

### PR DESCRIPTION
**Pull request by torokati44**
So the APKs uploaded as artifacts can be installed and tested.

### CODE REVIEW

**Comment by torokati44** [comment]
The main objective of signed APKs from PR runs is verified working.
Next:
 - Check that secrets are not leaked in logs (GH should protect us from this)
 - Check that APKs from runs within the repo itself are still signed and working

**Comment by torokati44** [comment]
Also, while we're here - would you mind replacing the keystore stored in repo secrets sometime with a "more official" one, @Dinnerbone ? :blush:

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -89,11 +89,21 @@ jobs:
         run: |
             echo $ENCODED_STRING | base64 -di > app/androidkey.jks
 
+      - name: Generate dummy keystore
+        if: ${{ github.event.pull_request.head.repo.fork }}
```
Comment (Dinnerbone) [inline .github/workflows/build.yml:93]: Can we rather do this if the secret is empty... That's the true thing you want, no?

**Review by Dinnerbone — DISMISSED** [review]

**Comment by Dinnerbone** [comment]
> Also, while we're here - would you mind replacing the keystore stored in repo secrets sometime with a "more official" one, @Dinnerbone ? :blush: 

At some point I will, but we shouldn't hold this PR up for it, I don't know when I'll next be available for development time

File: .github/workflows/build.yml (hunk 1 above)
Comment (torokati44) [reply .github/workflows/build.yml:93]: Makes sense!

Comment (torokati44) [reply .github/workflows/build.yml]: Changed.

Comment (torokati44) [reply .github/workflows/build.yml]: Unforty, not worky: 
```
Invalid workflow file: .github/workflows/build.yml#L93
The workflow is not valid. .github/workflows/build.yml (Line: 93, Col: 13): Unrecognized named-value: 'secrets'. Located at position 1 within expression: secrets.KEYSTORE == ''
```
Changed back.

Comment (Dinnerbone) [reply .github/workflows/build.yml:93]: You gotta set it to an env and then conditional on the env

eg: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-secrets

Comment (torokati44) [reply .github/workflows/build.yml]: Oooh, Aaah, I seee, thaanks!

**Comment by torokati44** [comment]
Eh, I want to test #108, so merging. I hope it will be fine, @Dinnerbone - feel free to amend it later as you wish, when you get the chance.

**Pull request merged**